usaspending_api/database_scripts/job_archive/backfill_unique_award_key.py

```diff
@@ -13,6 +13,10 @@
     Generates unique_award_keys for all transaction_fabs, transaction_fpds, and transaction_normalized records.
 
     SINGLE PROCESS VERSION
+
+    This script can be eliminated after sprint 81 has finished rolling out and
+    we are happy with the results.
+
```

Comment:
So...can we get rid of it now?

Reply:
Technically, yes, but I think we like to keep things around for a bit so we can remember how/why we did things more easily.  I just added this so we would know its lifespan at a glance (without having to bring up the ticket).